packages/web-components/src/checkbox/checkbox.template.ts

```diff
@@ -1,23 +1,24 @@
-import { ElementViewTemplate, html, slotted } from '@microsoft/fast-element';
-import { staticallyCompose } from '../utils/index.js';
-import type { Checkbox, CheckboxOptions } from './checkbox.js';
+import type { ElementViewTemplate } from '@microsoft/fast-element';
+import { html } from '@microsoft/fast-element';
+import { staticallyCompose } from '../utils/template-helpers.js';
+import type { Checkbox } from './checkbox.js';
+import type { CheckboxOptions } from './checkbox.options.js';
 
-const checkedIndicator = html.partial(`
-    <div class="checked-indicator">
-        <svg fill="currentColor"
-            aria-hidden="true"
-            width="1em"
-            height="1em"
-            viewBox="0 0 12 12"
-            xmlns="http://www.w3.org/2000/svg">
-                <path d="M9.76 3.2c.3.29.32.76.04 1.06l-4.25 4.5a.75.75 0 0 1-1.08.02L2.22 6.53a.75.75 0 0 1 1.06-1.06l1.7 1.7L8.7 3.24a.75.75 0 0 1 1.06-.04Z" fill="currentColor">
-                </path>
-        </svg>
-    </div>
+const checkedIndicator = html.partial(/* html */ `
+    <svg
+        fill="currentColor"
+        aria-hidden="true"
+        class="checked-indicator"
+        width="1em"
+        height="1em"
+        viewBox="0 0 12 12"
+        xmlns="http://www.w3.org/2000/svg">
+            <path d="M9.76 3.2c.3.29.32.76.04 1.06l-4.25 4.5a.75.75 0 0 1-1.08.02L2.22 6.53a.75.75 0 0 1 1.06-1.06l1.7 1.7L8.7 3.24a.75.75 0 0 1 1.06-.04Z" fill="currentColor"></path>
+    </svg>
 `);
 
-const indeterminateIndicator = html.partial(`
-    <div class="indeterminate-indicator"></div>
+const indeterminateIndicator = html.partial(/* html */ `
+    <div class="indeterminate-indicator" role="none"></div>
```

Comment:
`none` is similar to presentation, but I'm not sure if it is as well supported across AT's. Do you have any insight on that?

Reply:
I replaced the `<div>` with a `<span>`, which doesn't cause the "group" behavior.